test/fixtures/rules/std_rule_cases/L036.yml

```diff
@@ -153,7 +151,59 @@ test_distinct_single_fail_b:
     SELECT
       distinct a
     FROM my_table
-  # NOTE: This doesn't yet reflow nicely. We should solve that elsewhere.
   fix_str: |
     SELECT distinct a
-       FROM my_table
+    FROM my_table
+
+test_long_single_select_clause:
+  fail_str: "SELECT\n100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\n"
+
+test_single_select_with_comment_after_column:
+  fail_str: |
+    SELECT
+      1 -- this is a comment
+    FROM
+      my_table
+  fix_str: |
+    SELECT 1
+      -- this is a comment
+    FROM
+      my_table
+
+test_single_select_with_comment_after_column_no_space:
+  fail_str: |
+    SELECT
+      1-- this is a comment
+    FROM
+      my_table
+  fix_str: |
+    SELECT 1
+      -- this is a comment
+    FROM
+      my_table
+
+test_single_select_with_multiple_mixed_comments:
+  fail_str: |
+    SELECT
+      -- previous comment
+      1 -- this is a comment
+    FROM
+      my_table
+  fix_str: |
+    SELECT 1
+      -- previous comment
```

Comment:
Similar question as above -- should we consider moving the comments **above** the `SELECT`?

Reply:
Similar answer 😁